docs/Configuration.md

```diff
@@ -1,21 +1,32 @@
 # Browser and Platform Configuration <!-- omit in toc -->
 
-The browser and platform can be configured by two separate dictionaries. The
+The browser and platform can be configured by two separate classes. The
 platform configuration options can be set in `manager_params`, while the
 browser configuration options can be set in `browser_params`. The default
-settings are given in `openwpm/default_manager_params.json` and
-`openwpm/default_browser_params.json`.
+settings are given in `openwpm/config.py` in class `BrowserParams` and `ManagerParams`
 
-To load the default configuration parameter dictionaries we provide a helper
-function `TaskManager::load_default_params`. For example:
+To load the default configuration parameter just makes instances of `config::ManagerParams` and `config::BrowserPramas`. For example:
```

Comment:
It's generally considered bad writing to use words like "just" because it makes the reader feel like they should have known this and demotivates them.
Please remove it from this sentence.

Reply:
```suggestion
To load the default configuration create instances of `config::ManagerParams` and `config::BrowserParams`. For example:
```

Reply:
Good to know, committed!...